assets/src/blocks/Articles/ArticlePreview.js

```diff
@@ -97,95 +81,77 @@ export class ArticlePreview extends Component {
   render() {
     const {
       post: {
-        tags,
         link,
-        page_type,
-        page_type_link,
         post_title,
         post_excerpt,
         date_formatted,
         reading_time,
+        category,
       }
     } = this.props;
 
-    const articleClassName = tags.reduce((classname, tag) => classname + ` tag-${tag.slug}`, 'article-list-item');
```

Comment:
It's possible this is needed even when we don't output tags ourselves. Someone can create a class for a specific post and target any HTML element inside the `article`.

There's no harm in keeping these classes, they don't conflict with the style changes of this PR.

Reply:
It's depended on here, for example https://github.com/greenpeace/planet4-child-theme-switzerland/blob/3c27e85d541ed6f12d375658be96e5595d16a73f/src/scss/variables/_colors.scss#L27

Reply:
![Screenshot from 2022-03-25 16-52-00](https://user-images.githubusercontent.com/7604138/160155520-2378fc1e-14e8-4b26-af0b-898346528a74.png)
![Screenshot from 2022-03-25 16-51-31](https://user-images.githubusercontent.com/7604138/160155527-76d2a52c-7d09-4227-adf6-c25c81901c35.png)

Reply:
This is no longer relevant for this PR which focuses on the `See all` functionality